test/cypress/integration/web-payments/google-pay.test.cy.js

```diff
@@ -326,6 +329,15 @@ describe('Google Pay payment flow', () => {
       cy.log('Should not show Google Pay as payment is a recurring one')
 
       cy.get('#google-pay-payment-method-submit.web-payment-button--google-pay').should('not.exist')
+
+      cy.task('clearStubs')
+      cy.task('setupStubs', [...chargeStubsWithGooglePayOrApplePayEnabled(true, false, true, true)])
```

Comment:
Interesting that this is part of the same test rather than a separate one... You're just following the pattern that already existed though.

Reply:
yes, not sure why that was, but followed this format rather than refactor the whole thing